### DISCUSSION TITLE
Differentiate between fixes and infrastructure changes

### DISCUSSION THREAD

**Discussion by e-lo** [OvertureMaps/data#302] [Discussion]
## Problem
One of the difficulties in using current OSM data for looking at change-sets is the ability to differentiate between changes that occur IRL vs changes associated with filling in missing data or fixing errors.

This causes a lot of noise for change-sets and results in great difficulty in using OSM for identifying data such as:
- where new bicycle facilities were added;
- where new roadway capacity was added;
- where roadway speed limits were reduced, etc.

This in turn makes it very difficult for municipalities and researchers to understand the effects of changes on the transportation network on emissions, congestion, safety, etc.  This type of research and evidence or necessary in order to produce stable funding streams and make policy choices which benefit society.

Conversely, it has been quite straightforward to analyze change-sets for transit service and research their effect due to the nature of distinct service dates for GTFS datasets.

## Enhancements

I don't have a perfect proposed solution, but here are a few thoughts as to enhancements which could potentially help mitigate the above issue.

1.  require changes to be tagged as data enhancements vs infrastructure changes
2. provide traceability in the stable IDs when splitting segments (aka, this segment was previously known as ID=1234)
3. create/leverage a standard for `projects` which have a specific implementation date (see: [ProjectCards)](https://github.com/network-wrangler/projectcard)

**Comment by RobSoetewey-TomTom** [comment]
Hi @e-lo,

Thanks for your feedback.
The problem proposed here is interesting and far from trivial.

We do have your point 2. in focus and with GERS intend to bring considerable improvement there.

The other points require knowledge of intend for a change. We will not have access to that information for all our sources and processes, neither are we in a position to enforce that. It would be interesting to work out what can be derived e.g. from multi temporal satellite imagery. I'm not sure how to fit that in to our priorities at this point, but taking note of the want.

Rob

**Comment by e-lo** [comment]
>We will not have access to that information for all our sources 

Agree that current OSM data does not have this...and it is one reason why OSM model is flawed  😢 

There are definitely data producers who are interested in adding this data...somewhere.  

In thinking about if they could produce this as an add-on dataset,  perhaps understanding how the change-sets for Overture data will be produced would highlight if this is possible.  

I guess most importantly: 
 - Will there be IDs associated with *changes* to individual GERS or linear-referenced properties ? (aka - can they be referenced).

**Comment by vcschapp** [comment]
> There are definitely data producers who are interested in adding this data...somewhere.

This is interesting to know, as perhaps we could figure out a way to expose it where the data source provides it.

@e-lo Are you able to share any data producers offhand who are including such data, or who have at least have the data and have expressed a desire to share it?

**Comment by e-lo** [comment]
Metropolitan Planning Organizations (MPOs), which cover any region with 50k+ population, keep data about every "project" which has been planned or implemented using federal funding (most projects of significance) using a variety of formats and database structures saved locally.  Several MPOs are working together to create a data standard for infrastructure projects (see: https://github.com/Metropolitan-Council/project_card_registry - currently empty, but data exists locally , https://github.com/network-wrangler/projectcard).
